addons/cilium/cilium_v1.12.yaml

```diff
@@ -1069,6 +1074,7 @@ spec:
         - name: cilium-config-path
           mountPath: /tmp/cilium/config-map
           readOnly: true
+      terminationMessagePolicy: FallbackToLogsOnError
```

Comment:
Wrong indentation - please add 2 extra spaces in front of the `terminationMessagePolicy`